pkg/subctl/cmd/verify.go

```diff
@@ -152,9 +152,12 @@ func configureTestingFramework(args []string) {
 	framework.TestContext.ReportPrefix = "subctl"
 	framework.TestContext.SubmarinerNamespace = submarinerNamespace
 
-	// For some tests this is only printing, but in some of them they need those to be
-	// the cluster IDs that will be registered in the Cluster CRDs by submariner
-	framework.TestContext.ClusterIDs = []string{"ClusterA", "ClusterB"}
+	// Read the cluster names from the given kubeconfigs
+	rawConfigA, err := getClientConfig(args[0], "").RawConfig()
+	exitOnError("Error connecting to the first target cluster", err)
+	rawConfigB, err := getClientConfig(args[1], "").RawConfig()
+	exitOnError("Error connecting to the second target cluster", err)
+	framework.TestContext.ClusterIDs = []string{*getClusterName(rawConfigA), *getClusterName(rawConfigB)}
```

Comment:
Perhaps we should check nil return from `getClusterName` to be complete? ie:

```
func clusterFromConfig(kubeConfigPath string) string {
    rawConfig, err := getClientConfig(kubeConfigPath, "").RawConfig()
    exitOnError(fmt.Sprintf("Error obtaining the kube config for path %q", kubeConfigPath), err)
    cluster := getClusterName(rawConfig) 
    if cluster == nil {
        exitWithErrorMsg(fmt.Sprintf("Could not obtain the cluster name from kube config: %#v", rawConfig))
    }

    return *cluster
} 
```

Reply:
Done, thanks!